lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -1240,21 +1288,71 @@ apos.define('apostrophe-schemas', {
     self.addFieldType({
       name: 'select',
       populate: function(data, name, $field, $el, field, callback) {
-        var $options = $field.find('option');
-        // Synthesize options from the choices in the schema, unless
-        // the frontend developer has chosen to do it for us
-        if (!$options.length) {
-          _.each(field.choices, function(choice) {
-            var $option = $('<option></option>');
-            $option.text(choice.label);
-            $option.attr('value', choice.value);
-            $field.append($option);
+        var choices;
+        return async.series([
+          getChoices,
+          renderChoices
+        ], callback);
+        function getChoices(callback) {
+          if ((typeof field.choices) !== 'string') {
+            choices = field.choices;
```

Comment:
We don't do anything with `choices`, so the linter is angry. The `var choices;` on line 1291 is then also nada. I think we should be able to just nix this line and move on. Yes?